src/open_inwoner/conf/base.py

```diff
@@ -661,6 +664,36 @@
 
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 
+#
+# CELERY - async task queue
+#
+CELERY_BROKER_URL = os.getenv("CELERY_BROKER_URL", "redis://localhost:6379/0")
+# CELERY_RESULT_BACKEND = os.getenv("CELERY_RESULT_BACKEND", "redis://localhost:6379/0")
+
+# Add (by default) 5 (soft), 15 (hard) minute timeouts to all Celery tasks.
+# CELERY_TASK_TIME_LIMIT = config("CELERY_TASK_HARD_TIME_LIMIT", default=15 * 60)  # hard
+# CELERY_TASK_SOFT_TIME_LIMIT = config(
+#     "CELERY_TASK_SOFT_TIME_LIMIT", default=5 * 60
+# )  # soft
+
+CELERY_BEAT_SCHEDULE = {
+    "dummy_wait_task": {
```

Comment:
Lets perform the zgw_import_data task every morning at 07:00 to replace the cronjob